### PR TITLE
Enhance UG and fix bugs

### PR DESCRIPTION
**Pull request by limjierui**
Enhance list command in UG.

Fix list command to be able to take in a date.

### CODE REVIEW

File: data.txt [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # BEGIN EXPENDITURES
+ "amount": "5.0"  "description": "chicken rice"  "paymentMethod": "Cash"  "time": "12/03/2022"  "category": "Food"
```
Comment (SeanHoWB) [inline data.txt:2]: delete the data pls!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ accurate.
 Words in `[SQUARE_BRACKETS]` are the parameters. Words starting with a `/` are flags
 
 > **💡 Note:**
->- Parameters and flags are space-separated. For example: `list/e` is not a valid command while `list /e` is valid.
+>- All parameters are compulsory unless stated in the notes.
```
Comment (SeanHoWB) [inline docs/UserGuide.md:104]:
```suggestion
>- Parameters and flags are space-separated. For example: `list/e` is not a valid command while `list /e` is valid.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ accurate.
...
 
 > **⚠️Warning⚠️**
 >- All parameters are compulsory! Input the parameters in the order shown, or the application will not be able to read your
```
Comment (SeanHoWB) [inline docs/UserGuide.md:107]:
```suggestion
>- All parameters are compulsory unless stated in the notes.
>- Input the parameters in the order shown, or the application will not be able to read your
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -221,15 +250,16 @@ Use the `list /e` command to view the indices of your expenditures.
...
+    * Format of the date is `dd/mm/yyyy`.
     * For example: `12 March 2022` will be entered as `12/03/2022`.
 
 For example: `update /e 1 /pm cash /c Food /d chicken rice /a 5 /t 12/03/2022`.  
 Updates the first expenditure in your list to a $5.0 expenditure on food item 'chicken rice' that was paid in cash on 12
 March 2022.
 
 #### Expected Outcome:
-
+Update the first expenditure in your list to a $5.0 expenditure on food item 'chicken rice' that was paid in cash on 12
```
Comment (SeanHoWB) [inline docs/UserGuide.md:261]: why include this here when its mentioned above in line 257?

File: src/main/java/seedu/mindmymoney/command/ListCommand.java [hunk 5]
```diff
@@ -169,9 +161,9 @@ public void printExpenditureList() throws MindMyMoneyException {
             throw new MindMyMoneyException(
                     "Your expenditure list is currently empty! Please add some expenditures to your list first");
         } else {
-            System.out.print(PrintStrings.LINE);
+//            System.out.print(PrintStrings.LINE);
             System.out.print(expenditureListToString());
-            System.out.println(PrintStrings.LINE);
+//            System.out.println(PrintStrings.LINE);
```
Comment (SeanHoWB) [inline src/main/java/seedu/mindmymoney/command/ListCommand.java]: why commented?

**Review by SeanHoWB — COMMENTED** [review]
overall good! I left some comments

File: src/main/java/seedu/mindmymoney/command/ListCommand.java (hunk 5 above)
Comment (limjierui) [reply src/main/java/seedu/mindmymoney/command/ListCommand.java]: removed during gradle checks.

File: data.txt (hunk 1 above)
Comment (limjierui) [reply data.txt:2]: ok

File: docs/UserGuide.md (hunk 2 above)
Comment (limjierui) [reply docs/UserGuide.md:104]: resolved

File: docs/UserGuide.md (hunk 4 above)
Comment (limjierui) [reply docs/UserGuide.md:261]: resolved.

File: docs/UserGuide.md (hunk 3 above)
Comment (limjierui) [reply docs/UserGuide.md:107]: resolved.

**Pull request merged**